desktop/appcontainer/AppContainer.js

```diff
@@ -55,6 +56,10 @@ export class AppContainer extends Component {
 
     static modelClass = AppContainerModel;
 
+    static propTypes = {
+        model: PT.oneOfType([PT.instanceOf(AppContainerModel), PT.object]).isRequired
+    };
```

Comment:
Can remove this one - the app developer doesn't need to (and indeed can't) pass a model or object to this component.  Instead they provide the *class* within the config passed to `XH.renderApp()` and the toolkit ends up instantiating it with an already-created `AppContainerModel`.  

This is the first example here of where I think having the prop-type doc makes it seem as though you should expect to instantiate the component with props, which ends up being more misleading than helpful.